thrift/compiler/generate/t_php_generator.cc

```diff
@@ -1260,7 +1266,7 @@ void t_php_generator::_generate_php_struct_definition(
         !(t->is_struct() || t->is_xception())) {
       dval = render_const_value((*m_iter)->get_type(), (*m_iter)->get_value());
     }
-    indent(out) << "public $" << (*m_iter)->get_name() << " = " << dval << ";"
+    indent(out) << "private $" << (*m_iter)->get_name() << " = " << dval << ";"
```

Comment:
This would require a migration from all users. Keep this public for some time.

Reply:
Understood.